portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +53,24 @@ function rotateImages() {
   
   setTimeout("rotateImages()", time);
 }
+
+function getCommentsUsingArrows() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    commentsList = document.getElementById('comment-container');
+    commentsList.innterHTML = '';
```

Comment:
This line of code will execute, but it won't do what you're expecting.

`innerHTML` is a special property that affects what is rendered in the browser.
`innterHTML` (as written here) has no special behavior, but JavaScript will happily add it as a key to the `commentsList` object and assign it a value of `''`.

Unfortunately plain JavaScript doesn't do much to guarantee type safety or warn about uninitialized/undeclared fields, so you will have to test carefully.

Reply:
Thank you!